### PR TITLE
Django 3 + Python 3.6 support

### PR DESCRIPTION
**Pull request by mpasternak**
Hello @sveetch , long time no see!

here's a simple patch that takes `crispy-forms-foundation` to ANOTEHR LEVEL. 

* min Django supported: 2.0
* min Python supported: 3.6

Thank you for the review & eventual release.

### CODE REVIEW

File: crispy_forms_foundation/forms.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 from django import forms
 from django.forms.fields import FileField, ImageField
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```
Comment (sveetch) [inline crispy_forms_foundation/forms.py]: I didn't checked Django3 release changelog yet, "ugettext_lazy" has been removed ?

**Comment by mpasternak** [comment]
@sveetch , warning. This patch still needs work, I need to add layouts for "row.html" and "column.html" ...

**Comment by sveetch** [comment]
> @sveetch , warning. This patch still needs work, I need to add layouts for "row.html" and "column.html" ...

Ok bump here when you are done and ready for a review.

**Comment by mpasternak** [comment]
@sveetch The branch is ready for re-review. Missing files added in https://github.com/sveetch/crispy-forms-foundation/pull/43/commits/78759aa8e71daa90e8f2618e606761464fb7e7e8

**Comment by sveetch** [comment]
Started to review it and fixed some things in a new branch https://github.com/sveetch/crispy-forms-foundation/tree/mpasternak-master where the work will continue.

- Tox environment only test on Python 3.6
- Removed every occurences, statics and compat hacks for foundation5
- Updated tests

There is still some work to be done before releasing this like testing your row/column, etc..

**Pull request closed without merging**